buildmaster/ATLAS_2JET_7TEV_R06/filter.py

```diff
@@ -0,0 +1,329 @@
+# Filter for ATLAS_2JET_7TEV_R06
+"""
+Created on Mar  2023
+
+@author: Mark N. Costantini
+"""
+
+import yaml
+import numpy as np
+import pandas as pd
+from scipy.linalg import block_diag, cholesky
+
+
+from validphys.covmats import dataset_inputs_covmat_from_systematics
+from validphys.loader import Loader
+import warnings
+warnings.filterwarnings('ignore')
+
+
+
+def filter_ATLAS_2JET_7TEV_R06(scenario=0,kin_data=True):
+    """
+    Parameters
+    ----------
+
+    scenario : 0, 1, 2
```

Comment:
Could we have `Literal["nominal", "stronger", "weaker"]` as input instead? Would make the code a little bit nicer.